content/blog/1.0-announcement/index.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: Announcing GitOps 1.0
+date: "2021-10-11T09:00:01.284Z"
+description: "Open GitOps 1.0 brings a mature standard to the DevOps industry."
+author: Dan Garfield and ?
+thumbnail: images/
+---
+
+The GitOps Working Group has been working diligiently since announcing the formation of the group...
```

Comment:
```suggestion
The GitOps Working Group has been working diligently since announcing the formation of the group...
```

Fix minor typo: diligiently → diligently